exporters/otlp/src/main/java/io/opentelemetry/exporters/otlp/OtlpGrpcMetricExporter.java

```diff
@@ -146,12 +154,18 @@ public void shutdown() {
   /** Builder utility for this exporter. */
   public static class Builder extends ConfigBuilder<Builder> {
     private static final String KEY_METRIC_TIMEOUT = "otel.otlp.metric.timeout";
-
+    private static final String KEY_ENDPOINT = "otel.otlp.endpoint";
+    private static final String KEY_USE_TLS = "otel.otlp.use.tls";
+    private static final String KEY_METADATA = "otel.otlp.metadata";
     private ManagedChannel channel;
     private long deadlineMs = 1_000; // 1 second
+    private String endpoint = "localhost:55680";
```

Comment:
It might be more readable to have a constant defined, `DEFAULT_ENDPOINT`, `DEFAULT_DEADLINE_MS`

Reply:
👍 I opened #1465 to track